src/custom/components/Menu/index.tsx

```diff
@@ -162,6 +162,13 @@ export function Menu() {
           </span>
         </MenuItem>
 
+        <InternalMenuItem to="/play" onClick={close}>
+          <span role="img" aria-label="wizard-icon">
+            🐮
+          </span>
+          Game
+        </InternalMenuItem>
```

Comment:
Add a space before the text:
![screenshot_2021-05-28_08-05-06](https://user-images.githubusercontent.com/43217/120004317-75a46700-bf8b-11eb-8bbe-b29f60b2ccec.png)

Reply:
done